flexget/plugins/output/subtitles_subliminal.py

```diff
@@ -55,7 +55,7 @@ class PluginSubliminal:
           authentication:
             legendastv:
               username: myuser
-              passsword: mypassword
+              password: mypassword
```

Comment:
This can actually be removed altogether from the docs since legendas.tv no longer exists

Reply:
Removed `legendastv` and updated the example to use `opensubtitles`.